app/assets/stylesheets/okayNav.sass

```diff
@@ -0,0 +1,126 @@
+/*
+ * We are hiding the invisible nav outside the screen
+ * so we need to avoid the horizontal scroll
+
```

Comment:
Actually made no difference... I checked CSS in browser beforehand and the CSS of okayNav is actually rendered. However I wrote few end points just now.